### DISCUSSION TITLE
Kata 5

### DISCUSSION THREAD

**Discussion by RodrigoVega0** [LaunchX-InnovaccionVirtual/CursoIntroPython#119] [General]
Hola!! 

Estoy realizando la Kata 5, antes de empezar la kata, ingrese datos por teclado, al imprimir mi nombre me marca error, pero si ejecuto el mismo código en un archivo .py, lo ejecuta bien. Alguien sabe el por que?

![image](https://user-images.githubusercontent.com/99063020/153540376-f9a1ae13-fd44-4a76-b904-2cba09d279c4.png)

**Comment by Dfredude** [comment]
Hola Rodrigo, tienes ahi un simple **_syntax_** error:
Pusiste:
```python
print ("Hola " + nombre)
```
Forma correcta:
```python
print("Hola " + nombre)
```
Quita el espacio y ese error se quitara.
#### Espero te haya ayudado!

**Reply by JosephLeyva** [reply]
A lo mejor y usaste la palabra print como variable antes de ejecutar ese código. 
Es decir:
`print = "Algo...."`
Y al final usa la variable print en vez de la función por defecto de Python

**Reply by Dfredude** [reply]
@JosephLeyva `str object is not callable` significa que el compilador detecto que un str esta siendo llamado como funcion, lo cual lo rechaza.

**Comment by RodrigoVega0** [comment]
Intente como me aconsejo @Dfredude quitando el espacio y no ya intente de varias formas y no ejecuta bien. No tengo idea por que pasa. Lo ejecuto en un archivo .py y sin errores lo ejecuta. No se por que en el Notebook marca un error
![image](https://user-images.githubusercontent.com/99063020/153542199-0f9cfa6d-34cc-4136-9ca4-cf69cdc46e13.png)

**Reply by BarajasAngel** [reply]
Podrías utilizar otro método para mostrar la variable como por ejemplo los siguientes:
`print(f"Hola {nombre}")` 
`print("Hola  {0}".format(nombre))`

**Comment by EmmanuelMontes** [comment]
Tiene razón Joseph Leyva, la has de estar usando en otras celdas como variable y por eso 
![Capture](https://user-images.githubusercontent.com/43102305/153543835-7d836fde-5d60-4805-93da-d06dce00fcaf.PNG)

**Comment by RodrigoVega0** [comment]
Gracias por el apoyo compañeros. Solo tuve que reiniciar el Kernel y soluciono todo.
